src/components/Typography/README.md

```diff
@@ -0,0 +1,16 @@
+# \<Typography /\>
+
+### Typography is a helper component used to present contents clearly and uniformly
+
+By default, it renders a `<span>` tag which is controlled by the tag prop. The tag can be overridden with a valid html element or custom tag.
+
+**Props:**
```

Comment:
Minor: We shouldn't have to manually write props out.

We used to get these for free from PropTypes, but since we switched to TS I guess we don't.

I'd support leaving them out and creating an issue to figure out how to generate props documentation from TS.

But if you want to include them manually, that's fine too.

Reply:
Ok, I will remove them.